preprocessing.py

```diff
@@ -62,4 +75,6 @@ def deserializeIndex(filePath):
     # path = sys.argv[1]
     list_of_fields = ["TITLE"]
     preprocessing(path, list_of_fields)
-    print(deserializeIndex("all_terms_in_dict"))
+    all_terms = deserializeIndex("all_terms_in_dict")
+    sparse_index = creation_sparse_index(all_terms)
```

Comment:
I cannot understand for what reasons you create matrix containing all terms from index.
When I have wrote about creation of sparse matrices I mean that you should create such matrix for each document in your collection based on built terms index